source/packages/libraries/clients/assetlibrary-client/src/index.ts

```diff
@@ -25,5 +25,5 @@ export * from './client/search.service';
 export * from './client/profiles.model';
 export * from './client/profiles.service';
 
```

Comment:
we still need to export the other name as well. Assumption, if we don't have the other name referenced in this index file then it wont be exported.  Below is what i think we would need adding for backwards compatibility.
`
export {ASSETLIBRARY_CLIENT_TYPES, ASSTLIBRARY_CLIENT_TYPES} from './di/types';
`

Reply:
I made the change here as well. Thanks.